### ISSUE TITLE
Add `All` option

### ISSUE THREAD

**Issue by grosser** [djezzzl/database_consistency#125]
so I can do:

```
All:
  enabled: false

RedundantIndexChecker:
  enabled: true
```

**Comment by djezzzl** [comment]
Hi @grosser ,

I'm sorry it took so long to implement this feature. It was done today as part of: https://github.com/djezzzl/database_consistency/pull/127.

Please have a look at released version `1.2.0`.

**Comment by grosser** [comment]
awesome, thx!
